stl/src/stacktrace.cpp

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <format>
+#include <shared_mutex>
+#include <string>
+
+// clang-format off
+#include <Shlwapi.h>
+#include <Windows.h>
+#include <dbghelp.h>
+// clang-format on
+
+#pragma comment(lib, "Dbghelp.lib")

```

Comment:
Why not use https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/ntifs/nf-ntifs-rtlcapturestackbacktrace

Reply:
I do use it for capturing, although under `CaptureStackBackTrace` name, the DbgHelp APIs are still needed for function names/line names/line numbers.

Reply:
Yeah you're right

Reply:
[DIA SDK](https://docs.microsoft.com/en-us/visualstudio/debugger/debug-interface-access/debug-interface-access-sdk?view=vs-2022) could be another option, but I'm afraid of using COM in STL impl.